### PR TITLE
Add SECURITY.md policy

### PR DESCRIPTION
**Pull request by hoffie**
GPG details will be added via a future Issue/PR

Fixes jamulussoftware/jamuluswebsite#319

Note: This will be automatically linked from several places on Github. Example in my fork:
https://github.com/hoffie/jamulus/issues/new/choose
https://github.com/hoffie/jamulus/security/policy

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  Usernames can freely be chosen.
+- There is no encryption or message authentication.
+  Both protocol messages and audio are transmitted in the clear.
+  Anyone with access to the network traffic is able to listen to the audio or even modify it.
+- There is little protection against certain patterns of Denial of Service:
+  - Servers have a limited number of slots for clients which can easily be exhausted.
+  - There is no protection against spam in the chat function.
+  - Jamulus is UDP-based and UDP does not provide protection against source address spoofing. This enables all kinds of Denial of Service or Impersonation attacks.
+
+For some of these areas there are long-term ideas about extending Jamulus (e.g. usage of TCP, authentication), but they have not been decided upon or even implemented yet.
```
Comment (pljones) [inline SECURITY.md:37]: I'd hope "decided upon" happened _before_ "implemented".

Comment (hoffie) [reply SECURITY.md:37]: Agreed. Do you think the wording should be changed? At least the order is correct. :)

Comment (pljones) [reply SECURITY.md:37]: The wording implies that the implementation happens before the decision.

Comment (hoffie) [reply SECURITY.md]: Ok. I have removed the implementation part.

**Review by ann0see — APPROVED** [review]
Especially the limitations might need some further discussion. Nevertheless, this document looks good to me.

**Review by pljones — APPROVED** [review]

**Pull request merged**